tree/ntupleutil/v7/src/RNTupleInspector.cxx

```diff
@@ -181,6 +181,24 @@ ROOT::Experimental::RNTupleInspector::Create(std::string_view ntupleName, std::s
    return inspector;
 }
 
+std::string ROOT::Experimental::RNTupleInspector::GetCompressionSettingsAsString() const
+{
+   int algorithm = fCompressionSettings / 100;
+   int level = fCompressionSettings - (algorithm * 100);
+
+   std::string algorithmName;
+   switch (algorithm) {
+   case 1: algorithmName = "zlib"; break;
+   case 2: algorithmName = "LZMA"; break;
+   case 3: algorithmName = "old compression algorithm"; break;
```

Comment:
I guess the first letter could be capitalised here and in the default value
```suggestion
   case 3: algorithmName = "Old compression algorithm"; break;
```

Reply:
I wrote "zstd" in lowercase because that's how I initially thought it was written officially, but I now see Facebook also sometimes just capitalizes it so it doesn't really matter after all. I'll make it consistent with the rest!